src/server/controllers/sendChangePasswordEmail.ts

```diff
@@ -812,6 +812,31 @@ export const sendEmailInOkta = async (
 			error.status === 404 &&
 			error.code === 'E0000007'
 		) {
+			// if we're using passcodes, then show the email sent page with OTP input
+			// even if the user doesn't exist
+			if (passcodesEnabled && usePasscodesResetPassword) {
+				// set the encrypted state cookie to persist the email and stateHandle
+				// which we need to persist during the passcode reset flow
+				setEncryptedStateCookie(res, {
+					email,
+					stateHandle: `${crypto.randomUUID()}`, // use a random non-existent stateHandle, so encrypted value is different
```

Comment:
```suggestion
					stateHandle: `${crypto.randomUUID()}`, // use a random non-existent stateHandle, so encrypted value is different
```

generate a random string which is the same length as a real state handle